Contribute/how-to-write-links.md

```diff
@@ -68,20 +68,16 @@ A file link is used to link from one article to another within the docset.
 > [!NOTE]
 > None of the previous examples use the `~/` as part of the link. To link to an absolute path that begins at the root of the repository, start the link with `/`. Including the `~/` produces invalid links when navigating the source repositories on GitHub. Starting the path with `/` resolves correctly.
 
-### Structure of links on docs.microsoft.com
+### Structure of links on Microsoft Docs
 
-Content published on docs.microsoft.com has the following URL structure:
+Content published on Microsoft Docs has the following URL structure:
 
-```
-https://docs.microsoft.com/<locale>/<product-service>/[<feature-service>]/[<subfolder>]/<topic>[?view=<view-name>]
-```
+`https://docs.microsoft.com/<locale>/<product-service>/[<feature-service>]/[<subfolder>]/<topic>[?view=<view-name>]`
 
 Examples:
 
-```
-https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview
-https://docs.microsoft.com/en-us/powershell/azure/overview?view=azurermps-5.1.1
-```
+`https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview
```

Comment:
Why not use the multi-line code fencing here?

Reply:
Reverted :-)